astronomer/providers/snowflake/hooks/snowflake.py

```diff
@@ -101,11 +101,9 @@ async def get_query_status(self, query_ids: List[str]):
                                 "type": "FAILED_WITH_ERROR",
                                 "query_id": query_id,
                             }
-                        else:
-                            return status
                     return {"status": "success", "query_ids": sfqid}
             except ProgrammingError as err:
-                error_message = "Programming Error: {0}".format(err)
+                error_message = "Programming Error: {0}".format(str(err))
```

Comment:
In OSS airflow the convention is to use f string instead of format. The details can be found here: [https://realpython.com/python-f-strings/](https://realpython.com/python-f-strings/)